doc/whats-new.rst

```diff
@@ -8,6 +8,13 @@ v0.6.0 (unreleased)
 
 .. _whats-new.0.6.0:
 
+
+Documentation
+~~~~~~~~~~~~~
+- Updated Realistic Data examples in `Transforming Vertical Coordinates <https://xgcm.readthedocs.io/en/latest/transform.html>`_ (:pull:`322`)
```

Comment:
This looks great in the rendered version, but could you move it down under `v0.5.2`, since this change is already included in this version (which I will release immediately after merging this).

Reply:
Alright! Hope I did it correctly, please let me know.